graphdatascience/tests/integration/conftest.py

```diff
@@ -48,7 +48,7 @@ def gds() -> GraphDataScience:
 
 @pytest.fixture(scope="package")
 def gds_with_tls(request: FixtureRequest) -> GraphDataScience:
-    test_dir = os.path.dirname(request.path)
+    test_dir = os.path.dirname(request.path)  # type: ignore
```

Comment:
Maybe we can also use the approach that you suggested on the other PR. But it's also fine to ignore the warning.

Reply:
I guess that would be easier. We should probably update the mypy in the docker image though